samples/Azure.Management.Storage/Generated/StorageAccountContainer.cs

```diff
@@ -14,5 +16,8 @@ public partial class StorageAccountContainer
         protected StorageAccountContainer()
         {
         }
+
+        /// <summary> Gets the valid resource type for this object. </summary>
+        protected ResourceType ValidResourceType => "any";
```

Comment:
Create a bug to follow up on "any" as a valid resource type this shouldn't be here

Reply:
Added an extra acceptance criteria [here](https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5405) to handle this.